lib/api/api1.js

```diff
@@ -394,6 +414,15 @@ function searchQuery(request, cache, error) {
       query.availability = v;
   }
 
+  // check for invalid criteria
+  if (strict) {
```

Comment:
This sort of comment is useful enough it should go in the code itself rather than getting buried in a PR that will be closed out shortly
```suggestion
  // Optional error check for invalid criteria. This is only applicable for the
  // search verb since it gives false positives for other verbs that use search,
  // but also have other properties in the input.
  if (strict) {
```